src/AppBundle/Controller/UserController.php

```diff
@@ -73,4 +74,19 @@ public function avatarAction($username, Request $request)
 
         return $response;
     }
+
+    /**
+     * @Route("/register/resend-email", name="register_resend_email")
+     * @Template()
+     */
+    public function resendRegistrationEmailAction(Request $request, UserManagerInterface $userManager, MailerInterface $mailer)
+    {
+        $email = $request->request->get("email");
+        $user = $userManager->findUserByEmail($email);
```

Comment:
The variable `$email` may contain the email of a non-existing user.
You should make sure that `$user` is not `NULL`.